tests/utils/test_mime_types.py

```diff
@@ -16,6 +16,7 @@
     # not supported by doc dl
     ('test.pptx', 'application/vnd.openxmlformats-officedocument.presentationml.presentation'),
     ('test.zip', 'application/zip'),
+    ('test.xlsx', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'),
```

Comment:
Should this be in the support by doc dl section?

Reply:
Oh yes, good spot. I wasn't expecting the test to check for unsupported things.

Moved it 👍.